client/my-sites/domains/domain-management/contacts-privacy/card.jsx

```diff
@@ -110,29 +137,49 @@ class ContactsPrivacyCard extends React.PureComponent {
 			);
 		}
 
+		let privacyText = translate(
+			'{{strong}}Privacy Protection{{/strong}} is turned {{strong}}off{{/strong}} for this domain. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( '{{strong}}Privacy Protection{{/strong}} is turned off for this domain. Your contact information is {{strong}}public{{/strong}}. {{a}}Enable Privacy Protection{{/a}}' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).